src/Mpdf.php

```diff
@@ -1546,10 +1546,19 @@ public function __construct(array $config = [])
 		$this->selectoption = [];
 	}
 
-	public function cleanup()
+	public function cleanup($allReaders = false)
```

Comment:
I'm not sure about the parameter. Let's assume, for this case, it to be always false so that the method signature is not changed.

So please, remove the parameter and change the code accordingly.

Reply:
ok, removed and pushed, hope the tests don't throw me curve ball again :)